### DISCUSSION TITLE
Avatar Fontsize is not working!

### DISCUSSION THREAD

**Discussion by Maharajankannan** [chakra-ui/chakra-ui#8663] [General]
When i try to add the fontSize in the Avatar why am i not getting the desired fontSize
<Avatar name={name} fontSize='14px' `/>`

Is there any better solutions ?

**Comment by segunadebayo** [comment]
`fontSize` on `Avatar` does not style the initials. Put it on `Avatar.Fallback`:

```tsx
<Avatar.Root>
  <Avatar.Fallback name={name} fontSize="14px" />
</Avatar.Root>
```
